gradio/networking.py

```diff
@@ -35,8 +36,13 @@ def install_signal_handlers(self):
     def run_in_thread(self):
         self.thread = threading.Thread(target=self.run, daemon=True)
         self.thread.start()
+        start = time.time()
         while not self.started:
             time.sleep(1e-3)
+            if time.time() - start > 5:

```

Comment:
It takes <10 ms on my computer to start the server. So 5 seconds is a very generous allotment for slow computers.

Reply:
nit: Might be good to add "to start within 5 seconds." for more context. Will push this up since you're away

Reply:
Actually this message is not displayed to the user so won't make the change